<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FMainWindow</class>
 <widget class="QMainWindow" name="FMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_back"/>
   <addaction name="action_save"/>
   <addaction name="action_open"/>
   <addaction name="action_accept"/>
   <addaction name="action_man"/>
   <addaction name="action_woman"/>
   <addaction name="action_delete"/>
   <addaction name="action_trash"/>
   <addaction name="action_webcam"/>
   <addaction name="action_run"/>
   <addaction name="action_info"/>
   <addaction name="action_settings"/>
   <addaction name="action_help"/>
   <addaction name="action_fullScrean"/>
   <addaction name="actions_howMin"/>
  </widget>
  <action name="action_back">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/left.png</normaloff>:/icons/left.png</iconset>
   </property>
   <property name="text">
    <string>back</string>
   </property>
   <property name="toolTip">
    <string>back to prev</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/download.png</normaloff>:/icons/download.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/cross.png</normaloff>:/icons/cross.png</iconset>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_man">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/man.png</normaloff>:/icons/man.png</iconset>
   </property>
   <property name="text">
    <string>man</string>
   </property>
   <property name="toolTip">
    <string>show only man</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_woman">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/woman.png</normaloff>:/icons/woman.png</iconset>
   </property>
   <property name="text">
    <string>woman</string>
   </property>
   <property name="toolTip">
    <string>show only woman</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_webcam">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/camera.png</normaloff>:/icons/camera.png</iconset>
   </property>
   <property name="text">
    <string>webcam</string>
   </property>
   <property name="toolTip">
    <string>run camera</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="action_trash">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/trash.png</normaloff>:/icons/trash.png</iconset>
   </property>
   <property name="text">
    <string>trash</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="action_info">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/statistics.png</normaloff>:/icons/statistics.png</iconset>
   </property>
   <property name="text">
    <string>info</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_run">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/run.png</normaloff>:/icons/run.png</iconset>
   </property>
   <property name="text">
    <string>run</string>
   </property>
   <property name="toolTip">
    <string>run detection and recognition</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>settings</string>
   </property>
  </action>
  <action name="action_help">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
   <property name="toolTip">
    <string>add images</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_accept">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/accept.png</normaloff>:/icons/accept.png</iconset>
   </property>
   <property name="text">
    <string>accept</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </action>
  <action name="action_fullScrean">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/monitor.png</normaloff>:/icons/monitor.png</iconset>
   </property>
   <property name="text">
    <string>fullScrean</string>
   </property>
   <property name="shortcut">
    <string>Alt+F12</string>
   </property>
  </action>
  <action name="actions_howMin">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/monitor2.png</normaloff>:/icons/monitor2.png</iconset>
   </property>
   <property name="text">
    <string>showMin</string>
   </property>
   <property name="shortcut">
    <string>Alt+F12</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_fullScrean</sender>
   <signal>triggered()</signal>
   <receiver>FMainWindow</receiver>
   <slot>showFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actions_howMin</sender>
   <signal>triggered()</signal>
   <receiver>FMainWindow</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
